README.md

```diff
@@ -48,3 +48,24 @@ dependencies: [
 ```
 
 For detailed usage and API documentation, check [the documentation](https://swiftpackageindex.com/apple/swift-certificates/main/documentation/x509).
+
+## Benchmarks
+
+Benchmarks for `swift-certificates` are in a separate Swift Package in the `benchmarks` subfolder of this repository. 
+They use the [`package-benchmark`](https://github.com/ordo-one/package-benchmark) plugin.
+Benchmarks depends on the [`jemalloc`](https://jemalloc.net) memory allocation library, which is used by `package-benchmark` to capture memory allocation statistics.
+An installation guide can be found in the [Getting Started article](https://swiftpackageindex.com/ordo-one/package-benchmark/1.11.1/documentation/benchmark/gettingstarted#Installing-Prerequisites-and-Platform-Support) of `package-benchmark`. 
+Afterwards you can run the benchmarks from CLI by going to the `benchmarks` subfolder (e.g. `cd benchmarks`) and invoking:
+```
+swift package benchmark
+```
+
+Profiling benchmarks in Xcode (or building the benchmarks in release mode) with `jemalloc` is currently not supported and requires disabling `jemalloc`. 
+Make sure Xcode is closed and then open it from the CLI with the `BENCHMARK_DISABLE_JEMALLOC=true` environment variable set e.g.:
+```
+BENCHMARK_DISABLE_JEMALLOC=true xed .
+```
```

Comment:
Do they have this documented as well? I would prefer if we just keep as much out of our docs as possible.

Reply:
No they haven't. I'm fine to remove it if `package-benchmark` adds it but we can also remove it again in a subsequence PR to unblock more benchmarks that I have in the pipeline. cc @hassila

Reply:
We can definitely add the comment about how to run in Xcode for profiling etc, np.